src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -37,7 +37,7 @@ public FindCommand(Set<String> keywords) {
     @Override
     public CommandResult execute() {
         final List<ReadOnlyPerson> personsFound = getPersonsWithNameContainingAnyKeyword(keywords);
-        return new CommandResult(getMessageForPersonListShownSummary(personsFound), personsFound);
+        return new CommandResult(getMessageForPersonListShownSummary(personsFound, "listed"), personsFound);
     }
```

Comment:
Also, due to passing in string in this manner, it is a violation of the coding standard of Magic Number (or Magic String in this case)